src/Native/HMAC.pack.js

```diff
@@ -0,0 +1 @@
+window.Crypto = require("crypto-js");
```

Comment:
I guess that the node tester provides a browser-like environment in which to run the tests.